com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UploadAppStep.java

```diff
@@ -120,7 +122,8 @@ private void uploadFiles(DelegateExecution context, final CloudFoundryOperations
         FileDownloadProcessor uploadFileToControllerProcessor = new DefaultFileDownloadProcessor(StepsUtil.getSpaceId(context),

```

Comment:
These methods look very similar. Isn't there a way to remove this code duplication?

Reply:
Leave it as it is since there is a reason why there is code duplication :)

Reply:
No this is not refactoring task. Consider open refactoring backlog about this.